### PR TITLE
support laravel 10 and php 8.1

### PR DESCRIPTION
**Pull request by fh32000**
@romanzipp 
This PR for #107

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,19 +11,19 @@
         }
     ],
     "require": {
-        "php": "^7.2|^8.0",
+        "php": "^7.2|^8.0|^8.1",
```
Comment (romanzipp) [inline composer.json:14]: There is no need to add the `^8.1` since it's included in `^8.0`

File: composer.json [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
-        "illuminate/support": "^5.5|^6.0|^7.0|^8.0|^9.0",
+        "illuminate/database": "^5.5|^6.0|^7.0|^8.0|^9.0|^10.0",
+        "illuminate/queue": "^5.5|^6.0|^7.0|^8.0|^9.0|^10.0",
+        "illuminate/support": "^5.5|^6.0|^7.0|^8.0|^9.0|^10.0",
         "nesbot/carbon": "^2.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.0",
-        "laravel/framework": "^5.5|^6.0|^7.0|^8.0|^9.0",
+        "laravel/framework": "^10.0",
```
Comment (romanzipp) [inline composer.json:24]: This will break existing tests

File: composer.json [hunk 3]
```diff
@@ -50,5 +50,7 @@
     },
     "config": {
         "sort-packages": true
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```
Comment (romanzipp) [inline composer.json:55]: Why raise min stability?

**Comment by romanzipp** [comment]
I'll add L10 support later today in a separate PR

**Comment by fh32000** [comment]
Thanks for reviewing the code. I agree with you . waiting for update

**Pull request closed without merging**